style.css

```diff
@@ -0,0 +1,122 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  background-color: #f1f2f3;
+  margin: 0;
+  font-family: sans-serif;
+}
+.col1 {
+  width: 59%;
+  height: 100vh;
+  display: inline-block;
+  position: relative;
+}
+.col1-text {
+  border: none 1px solid;
+  text-align: left;
+  position: absolute;
+  right: 0;
+  top: 30vh;
+}
+.text-small {
+  font-size: 0.8rem;
+}
+.text-margin-top-small {
+  margin-bottom: 0px;
+}
+ul {
+  list-style: none;
+}
+li {
+  margin-top: 16px;
+}
+.col2 {
+  width: 40%;
+  height: 100vh;
+  overflow: none;
+  display: inline-block;
+}
+.button-container {
+  width: 300px;
+  padding-left: 10px;
+  position: absolute;
```

Comment:
In fact, you can remove styles for .button-container entirely after applying the suggestion with col2-context